src/pages/Sale/SalePage.tsx

```diff
@@ -1,7 +1,105 @@
+    photo: "https://images.unsplash.com/photo-1585487000160-6ebcfceb0d03?auto=format&fit=crop&q=60&w=800&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MTV8fGNsb3RoaW5nfGVufDB8fDB8fHww",
+    user: "Dom",
+    location: "Toronto",
+    description: "I purchased these items (cost $678) after shoulder surgery to help control pain and swelling. The BREG cooling cube is filled with ice water, which circulates through the shoulder wrap. The wrap could also be used on elbows, knees or ankles. The shoulder brace immobilizes the shoulder while it heals."
+  },
+  {
+    id: 5,
+    title: "T-shirt",
+    price: 30,
+    photo: "https://images.unsplash.com/photo-1576566588028-4147f3842f27?auto=format&fit=crop&q=60&w=800&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MTl8fGNsb3RoaW5nfGVufDB8fDB8fHww",
+    user: "Ed",
+    location: "Toronto",
+    description: "I purchased these items (cost $678) after shoulder surgery to help control pain and swelling. The BREG cooling cube is filled with ice water, which circulates through the shoulder wrap. The wrap could also be used on elbows, knees or ankles. The shoulder brace immobilizes the shoulder while it heals."
+  },
+  {
+    id: 6,
+    title: "NIKE",
+    price: 30,
+    photo: "https://images.unsplash.com/photo-1543508282-6319a3e2621f?auto=format&fit=crop&q=60&w=800&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8OTR8fGNsb3RoaW5nfGVufDB8fDB8fHww",
+    user: "Frank",
+    location: "Toronto",
+    description: "I purchased these items (cost $678) after shoulder surgery to help control pain and swelling. The BREG cooling cube is filled with ice water, which circulates through the shoulder wrap. The wrap could also be used on elbows, knees or ankles. The shoulder brace immobilizes the shoulder while it heals."
+  },
+  {
+    id: 7,
+    title: "Hat",
+    price: 30,
+    photo: "https://images.unsplash.com/photo-1620231109648-302d034cb29b?auto=format&fit=crop&q=60&w=800&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MTExfHxjbG90aGluZ3xlbnwwfHwwfHx8MA%3D%3D",
+    user: "Gigi",
+    location: "Toronto",
+    description: "I purchased these items (cost $678) after shoulder surgery to help control pain and swelling. The BREG cooling cube is filled with ice water, which circulates through the shoulder wrap. The wrap could also be used on elbows, knees or ankles. The shoulder brace immobilizes the shoulder while it heals."
+  },
+]
+
 const SalePage = () => {
-    return (
-        <div>SalePage</div>
-    )
+  const navigate = useNavigate();
+  return (
+    <div>
+      <RowComponent style={{ marginBottom: 50 }}>
+        <span style={{ fontSize: 35, fontWeight: 700, fontStyle: "italic"}}>For Sale</span>
+        <Button title='Post a listing' />
+      </RowComponent>
+      <RowComponent style={{ display: "flex", flexWrap: "wrap", gap: 30, justifyContent: "flex-start" }}>
+        {
+          SaleItemsList.map(item => {
+            const { id, title, price, photo} = item;
+            return (
+              <div>
```

Comment:
`<div key={id}>`

to get rid of warning (Each child in a list should have a unique "key" prop.)